pkg/testing/yield_token_test.go

```diff
@@ -1,4 +1,4 @@
-package lendertesting
+package testing
```

Comment:
wut? was yield_token_test in the `lendertesting`? well it _should_ be in `testing` so all good...